src/error.rs

```diff
@@ -89,6 +89,12 @@ pub enum Error {
         message: bolt_proto::message::Message,
     },
 
+    // Returned if a bb8 connection cannnot be obtained form the pool
+    #[cfg(feature = "neo4j")]
+    Neo4jPoolGetConnectionFailed {
+        source: bb8::RunError<bb8_bolt::Error>,
```

Comment:
I would recommend also adding an `impl From<bb8:RunError<bb8_bolt::Error>> for Error` block below, for convenience and consistency with other error handling.

Reply:
Fixed.